.gitignore

```diff
@@ -25,3 +25,8 @@ npm-debug.log
 # WebStorm user-specific
 .idea/workspace.xml
 .idea/tasks.xml
+.idea/cesium.iml
+.idea/encodings.xml
+.idea/inspectionProfiles/Project_Default.xml
+.idea/jsLinters/jshint.xml
+.idea/misc.xml
```

Comment:
No problem, I'm not WebStorm expert so I wanted to make sure we didn't do something non-standard on our side. Most of us our running WebStorm 11, so if you're on an earlier version, that could be the source of it changing files locally.  Out of curiosity, what did it change?  Maybe we can prevent that from happening with a different config tweak.